core/src/main/java/com/graphhopper/routing/util/AbstractFlagEncoder.java

```diff
@@ -181,6 +181,14 @@ public int defineRelationBits(int index, int shift) {
         return shift;
     }
 
+    /**
+     * Decide whether a restriction relation is enabled for a given mode of travel. This skips some relations before
```

Comment:
No need to repeat the javadocs that are already in `FlagEncoder` here

Reply:
ok